lib/login/login.dart

```diff
@@ -122,6 +131,14 @@ class _LoginPageState extends State<LoginPage> {
           ),
         ),
       );
+      if (repoNumber != 200) {
+        // jumps to signup
+        Navigator.push(
+          context,
+          MaterialPageRoute(builder: (context) => SignUp(email: userEmail)),
```

Comment:
Instead of passing `userEmail` pass it from the response you get from userData

Reply:
@frlzjosh Can you clarify on userData['email']? 

I am passing userEmail to SignUp, so the user does not need to type in there email again.

Reply:
My mistake, I did not understand the whole workflow. This is fine